src/prepack-node.js

```diff
@@ -85,15 +53,16 @@ export function prepackFile(
     filename: string,
     options: Options = defaultOptions,
     callback: (any, ?{code: string, map?: SourceMap})=>void,
-    errorHandler?: (err: ?Error)=>void) {
+    fileErrorHandler?: (err: ?Error)=>void,
+    errorHandler?: ErrorHandler) {
```

Comment:
It would be idiomatic to put any optional arguments into the `options` objects. In fact, it's pretty common to put logging events that can be called more than once such as this one in there. E.g. as `onError` in the options.

We should also not have the `fileErrorHandler` here. File errors should pass the error to the first argument of the `callback`. This is idiomatic Node.js style that the callback of an async operation is the last argument. It should only be called once - if it completes or if it terminally fails.

This lets this work with this conversion to Promises in modern environments: http://2ality.com/2017/05/util-promisify.html Which is now idiomatic in Node 8.